modules/central-services-public-config.adoc

```diff
@@ -271,49 +274,72 @@ When you disable autoscaling, the `minReplicas` and `maxReplicas` parameters do
+| The CPU request for the StackRox Scanner to override the default value.
 
 | `scanner.resources.limits.memory`
-| The memory limit for Scanner to override the default value.
+| The memory limit for the StackRox Scanner to override the default value.
 
 | `scanner.resources.limits.cpu`
-| The CPU limit for Scanner to override the default value.
+| The CPU limit for the StackRox Scanner to override the default value.
 
 | `scanner.dbResources.requests.memory`
-| The memory request for Scanner database deployment to override the default values.
+| The memory request for the StackRox Scanner database deployment to override the default values.
 
 | `scanner.dbResources.requests.cpu`
-| The CPU request for Scanner database deployment to override the default values.
+| The CPU request for the StackRox Scanner database deployment to override the default values.
 
 | `scanner.dbResources.limits.memory`
-| The memory limit for Scanner database deployment to override the default values.
+| The memory limit for the StackRox Scanner database deployment to override the default values.
 
 | `scanner.dbResources.limits.cpu`
-| The CPU limit for Scanner database deployment to override the default values.
+| The CPU limit for the StackRox Scanner database deployment to override the default values.
 
 | `scanner.image.registry`
-| A custom registry for the Scanner image.
+| A custom registry for the StackRox Scanner image.
 
 | `scanner.image.name`
-| The custom image name that overrides the default Scanner image name (`scanner`).
+| The custom image name that overrides the default StackRox Scanner image name (`scanner`).
 
 | `scanner.dbImage.registry`
-| A custom registry for the Scanner DB image.
+| A custom registry for the StackRox Scanner DB image.
 
 | `scanner.dbImage.name`
-| The custom image name that overrides the default Scanner DB image name (`scanner-db`).
+| The custom image name that overrides the default StackRox Scanner DB image name (`scanner-db`).
 
 | `scanner.dbNodeSelector`
-| Specify a node selector label as `label-key: label-value` to force Scanner DB to only schedule on nodes with the specified label.
+| Specify a node selector label as `label-key: label-value` to force the StackRox Scanner DB to only schedule on nodes with the specified label.
 
 | `scanner.dbTolerations`
-| If the node selector selects tainted nodes, use this parameter to specify a taint toleration key, value, and effect for Scanner DB. This parameter is mainly used for infrastructure nodes.
+| If the node selector selects tainted nodes, use this parameter to specify a taint toleration key, value, and effect for the StackRox Scanner DB. This parameter is mainly used for infrastructure nodes.
 
 |===
 
+[id="central-services-public-configuration-file-scannerv4_{context}"]
+== Scanner V4
+Configurable parameters for Scanner V4.
+
+#Are these similar to the StackRox scanner parameters, but with "v4" in the name? I assume the descriptions would be the same.#
```

Comment:
Using [this as a reference](https://github.com/stackrox/stackrox/blob/master/image/templates/helm/shared/internal/scanner-v4-config-shape.yaml), believe the new values would be:

```
scannerV4.disable
scannerV4.exposeMonitoring
scannerV4.indexer.replicas
scannerV4.indexer.logLevel
scannerV4.indexer.nodeSelector
scannerV4.indexer.tolerations
scannerV4.indexer.autoscaling.disable
scannerV4.indexer.autoscaling.minReplicas
scannerV4.indexer.autoscaling.maxReplicas
scannerV4.indexer.resources.requests.memory
scannerV4.indexer.resources.requests.cpu
scannerV4.indexer.resources.limits.memory
scannerV4.indexer.resources.limits.cpu
scannerV4.matcher.replicas
scannerV4.matcher.logLevel
scannerV4.matcher.nodeSelector
scannerV4.matcher.tolerations
scannerV4.matcher.autoscaling.disable
scannerV4.matcher.autoscaling.minReplicas
scannerV4.matcher.autoscaling.maxReplicas
scannerV4.matcher.resources.requests.memory
scannerV4.matcher.resources.requests.cpu
scannerV4.matcher.resources.limits.memory
scannerV4.matcher.resources.limits.cpu
scannerV4.db.resources.requests.memory
scannerV4.db.resources.requests.cpu
scannerV4.db.resources.limits.memory
scannerV4.db.resources.limits.cpu
scannerV4.image.registry
scannerV4.image.name
scannerV4.db.image.registry
scannerV4.db.image.name
scannerV4.db.nodeSelector
scannerV4.db.tolerations
```